packages/flutter_tools/lib/src/asset.dart

```diff
@@ -400,8 +414,39 @@ class ManifestAssetBundle implements AssetBundle {
       }
     }
 
+    for (final DataAsset dataAsset in dartHookResult?.dataAssets ?? <DataAsset>[]) {
```

Comment:
Not in the isolated directory, so we can't use `DataAsset`s here. The `HookResult` should contain something else than `DataAsset` and `CodeAsset`, it should return the info in a format not requiring to import `package:native_assets_cli`.

Reply:
So we can't use `package:native_assets_cli` outside of `isolated`?

Reply:
No, you'll have to extract the info you need from `DataAsset`s inside `isolated/` and then make that info available here.